### PR TITLE
Support cold start gcmc

### PR DESCRIPTION
**Pull request by nishiba**

### CODE REVIEW

**Comment by nishiba** [comment]
@6syun9 @hirosassa @teppei-kanayama Please check my code. Thanks!!

File: sandbox/sandbox.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+def main():
+    x = np.array([0, 1, 2])
+    y = np.array([3, 4])
+    z = np.concatenate([x, y])
+    print(z)
+
+
+if __name__ == '__main__':
+    main()
```
Comment (hirosassa) [inline sandbox/sandbox.py]: [ASK] Is this needed?

Comment (nishiba) [reply sandbox/sandbox.py]: No, this is not needed. I delete this file.

**Comment by hirosassa** [comment]
Looks good to me!

**Comment by 6syun9** [comment]
LGTM!

**Pull request merged**